components/editor/modules/embed/EmbedLoader.js

```diff
@@ -79,9 +83,7 @@ export default (query, Component) => (
               data-active={active}
               contentEditable={false}
               >
-              <Component
-                data={node.data.toJS()}
-                />
+              {data.thumbnail && <Component data={data} />}
```

Comment:
Note that I still need this even with
https://github.com/orbiting/styleguide/commit/3590bfd53efdd0e3ffa5e01660e086ff9ca01204